tasks/main.yml

```diff
@@ -1,83 +1,82 @@
 ---
-- include: check_vars.yml
+- include_vars: "{{ item }}"
+  with_first_found:
+    - "../vars/{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
+    - "../vars/{{ ansible_distribution }}.yml"
+    - "../vars/{{ ansible_os_family }}.yml"
 
-- name: check if sox installed
-  command: "which {{ sox_install_dir }}/sox"
-  register: sox_exists
-  ignore_errors: yes
-  changed_when: False
-  failed_when: False
-  
+# Install Sox and libraries from package when enabled
+- include: install_package.yml
```

Comment:
I recommend to rename:

- `install_package.yml` to `install_from_package.yml`
- `install_sox.yml` to `install_from_source.yml`